src/siembol/resources/application.manager.properties

```diff
@@ -0,0 +1,37 @@
+management.endpoint.health.show-details=always
```

Comment:
@jgiannuzzi made a good point meybe we can add these default app properties into docker image in siembol CI/CD